bin/ex_libris_holdings_xml_parser.rb

```diff
@@ -0,0 +1,105 @@
+require 'nokogiri'
+
+class ExLibrisHoldingsXmlParser
+  def main(files)
+    yield HTRecord.header_tsv
+    files.each do |file|
+      File.open(file) do |io|
+        Nokogiri::XML(io).xpath("//record").each do |xml_record|
```

Comment:
Definitely fine as a first pass, but if we do this for real we should use rubymarc - if nothing else that parses the XML in streaming mode rather than trying to load the whole file into memory